integration-tests/src/test/java/com/datastax/dse/driver/api/core/cql/continuous/ContinuousPagingITBase.java

```diff
@@ -61,7 +62,9 @@ protected static void initialize(CqlSession session, DriverExecutionProfile slow
     }
     int count = 0;
     for (int i = 0; i < 200; i++) {
-      BatchStatement batch = BatchStatement.newInstance(DefaultBatchType.UNLOGGED);
+      BatchStatement batch =
+          BatchStatement.newInstance(DefaultBatchType.UNLOGGED)
+              .setConsistencyLevel(ConsistencyLevel.ALL);
```

Comment:
Which test failure this CL_ALL is trying to address?

Reply:
This was an experiement to attempt to fix flakiness where the queries in these tests fail to find all the items added in the test set-up. It wasn't supposed to be included in this PR so should be removed, thanks for catching this.